src/isar/config/logging.conf

```diff
@@ -21,16 +16,24 @@ handlers:
 loggers:
   console:
     level: DEBUG
-    handlers: [console]
+    handlers: []
     propagate: no
   api:
     level: DEBUG
-    handlers: [api, console]
+    handlers: [api]
```

Comment:
Why has the console handler been removed? Does the logging still log to the console or only to files now?